resources/qml/Dialogs/RenameDialog.qml

```diff
@@ -55,32 +59,58 @@ UM.Dialog
             id: nameField
             width: parent.width
             text: base.object
+            placeholderText: base.objectPlaceholder
+            placeholderTextColor: UM.Theme.getColor("text_field_text_disabled")
```

Comment:
I think you might be overriding the default color with the default color here.

Reply:
Can't hurt, and might help: More things are theme-able this way, especially with things like dark mode.